torch/testing/_internal/common_device_type.py

```diff
@@ -3,6 +3,8 @@
 import inspect
 import runpy
 import threading
+import typing
+from typing import Union, List, TypeVar, Tuple, Generic, ClassVar, Iterable, Any, Optional
```

Comment:
you can ignore the `import psutil` line exception by appending `# type: ignore[import]` for now in line 16